component/snackbar/snackbar.css

```diff
@@ -0,0 +1,29 @@
+.snackbar{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: fit-content;
+    padding: 0.5rem 2rem;
+    z-index: 5;
```

Comment:
I feel based on your screen, to add bit more depth to the box-shadow, so that it appears on top

Reply:
- also add some min-width so that you will see a gap between text and close icon, make close icon a bit smaller